vendors/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
```

Comment:
nit: we can delete this "empty" file

Reply:
Previously had to ignore the world for vendors